src/herbivoresAndCarnivores.js

```diff
@@ -1,40 +1,37 @@
 'use strict';
 
 class Animal {
-  constructor(health = 100, name) {
+  constructor(name, health = 100) {
     this.name = name;
-    this.name = health;
-
+    this.health = health;
     Animal.alive.push(this);
-  }
+  };
 }
 
 Animal.alive = [];
 
 class Herbivore extends Animal {
   constructor(name) {
     super(name);
-
     this.hidden = false;
   }
-
   hide() {
     this.hidden = true;
-  }
+  };
 }
 
 class Carnivore extends Animal {
-  bite(victim) {
-    const indexOfVictim = Animal.alive.indexOf(victim);
+  bite(animal) {
+    const indexOfAnimal = Animal.alive.indexOf(animal);
 
-    if (victim.hidden === true || victim instanceof Carnivore) {
+    if (animal.hidden === true || animal instanceof Carnivore) {
       return;
     }
 
-    victim.health -= 50;
+    animal.health -= 50;
```

Comment:
Have some doubts about this option of code writing, but assumt this code is fine too.